modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -5,9 +5,11 @@ endif::[]
 
 Mule execution failures result in an error. When a Mule app is running, both
 the Mule runtime engine and the module and connector operations configured
-in the app can throw errors.
+in the app can throw errors that are encapsulated within a Mule error object, which
+are associated with a Mule event. Mule applications log these errors and enable you
+to handle them with On-Error components. 
 
-Mule errors are grouped into <<error_types, error types>> that follow a hierarchical
+Mule error objects are grouped into <<error_types, error types>> that follow a hierarchical
```

Comment:
i would leave this as is, since the text describes the concept of an error, and not its representation.

Reply:
restored